packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -556,36 +554,33 @@ describe('src/cy/commands/querying', () => {
         })
       })
 
-      describe('route aliases', () => {
+      describe('intercept aliases', () => {
         it('returns the xhr', () => {
           cy
-          .server()
-          .route(/users/, {}).as('getUsers')
+          .intercept(/users/, {}).as('getUsers')
           .visit('http://localhost:3500/fixtures/jquery.html')
           .window().then({ timeout: 2000 }, (win) => {
             return win.$.get('/users')
-          }).get('@getUsers').then((xhr) => {
-            expect(xhr.url).to.include('/users')
+          }).wait('@getUsers').get('@getUsers').then((xhr) => {
```

Comment:
I was about to ask "wait, shouldn't this 'just work' now with the detached DOM work, until I remembered that it's not in the 12.0 branch yet. Hopefully soon this .wait() won't be necessary!

Or at least it won't if we switch to a chain of queries, like `.get('@getUsers').its('response.url').should('include', '/users')`. .then still won't retry / requery the subject in query-land.